render.yaml

```diff
@@ -17,6 +26,15 @@ services:
     startCommand: cd javascript && yarn && yarn start
```

Comment:
how come these are named "typescript" but then run in the javascript folder? we're going to eventually want to run both so we should standardize the names

Reply:
ugh. Should be `javascript`